site/components/pages/Resources/Container/index.tsx

```diff
@@ -0,0 +1,142 @@
+import Link from "next/link";
+import * as styles from "./styles";
+import { cx } from "@emotion/css";
+
+import {
+  PageWrap,
+  HeaderDesktop,
+  Footer,
+  NavItemDesktop,
+  HeaderMobile,
+  NavItemMobile,
+  ButtonPrimary,
+} from "@klimadao/lib/components";
+import { t, Trans } from "@lingui/macro";
+import ArrowBack from "@mui/icons-material/ArrowBack";
+
+import { urls } from "@klimadao/lib/constants";
+import { PageHead } from "components/PageHead";
+import { IS_PRODUCTION } from "lib/constants";
+import { FC, PropsWithChildren, ReactNode } from "react";
+
+type PageName = "blog" | "community" | "contact";
+
+export type Props = PropsWithChildren<ReactNode> & {
+  activePage: PageName;
+};
+
+export const Container: FC<Props> = (props) => {
+  const getListItemClasses = (name: PageName) => {
+    if (name === props.activePage) {
+      return cx(styles.listItem, styles.listItemActive);
+    }
+    return styles.listItem;
+  };
+
+  const _listItem = (name: PageName, textEl: ReactNode) => {
+    return (
+      <li className={getListItemClasses(name)}>
+        <Link href={`/resources/${name}`}>
+          <a>
+            {textEl}
+            {name === props.activePage && <ArrowBack />}
+          </a>
+        </Link>
+      </li>
+    );
+  };
+
+  return (
+    <>
+      <PageHead
+        production={IS_PRODUCTION}
+        title={t`Contact KlimaDAO`}
+        mediaTitle={t`Contact KlimaDAO`}
```

Comment:
these need to be different for each page

Reply:
🤦‍♂️